spec/model_spec.cr

```diff
@@ -97,4 +120,48 @@ describe LuckyRecord::Model do
       QueryMe.column_names.should eq [:id, :created_at, :updated_at, :email, :age]
     end
   end
+
+  describe ".ensure_correct_field_mappings" do
+    # table is missing
```

Comment:
I think you can remove these comments since the `it` blocks does a good job of explaining it

Reply:
ah yes I forgot to remove these